HyperVExtension/src/HyperVExtension/HyperVExtension.csproj

```diff
@@ -44,14 +44,14 @@
     If building from the build script the zip file should exist as it's created as part of the script execution.
     VS currently doesn't have a task to create it and it may not exist if it wasn't previously created by the build script.
     We will not treat that as an error for now.
-    If building Dev Home for x64 or x86 paltforms DevSetupAgent will be built only as x86 to reduce package size.
+    If building Dev Home for x64 or x86 platforms DevSetupAgent will be built only as x86 to reduce package size.
     x86 binaries can be deployed on both x64 and x86 VMs.
     -->
   <ItemGroup>
-    <None Include="..\DevSetupAgent\bin\DevSetupAgent_x86.zip" Condition="('$(Platform)'=='x64' or '$(Platform)'=='x86') and ('$(BuildingInsideVisualStudio)' != 'True' or Exists('..\HyperVExtension\src\DevSetupAgent\bin\DevSetupAgent_x86.zip'))">
+    <None Include="..\DevSetupAgent\bin\x86\$(Configuration)\DevSetupAgent_x86.zip" Condition="('$(Platform)'=='x64' or '$(Platform)'=='x86') and ('$(BuildingInsideVisualStudio)' != 'True' or Exists('..\HyperVExtension\src\DevSetupAgent\bin\x86\($Configuration)\DevSetupAgent_x86.zip'))">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Include="..\DevSetupAgent\bin\DevSetupAgent_arm64.zip" Condition="'$(Platform)'=='arm64' and ('$(BuildingInsideVisualStudio)' != 'True' or Exists('..\HyperVExtension\src\DevSetupAgent\bin\DevSetupAgent_arm64.zip'))">
+    <None Include="..\DevSetupAgent\bin\arm64\($Configuration)\DevSetupAgent_arm64.zip" Condition="'$(Platform)'=='arm64' and ('$(BuildingInsideVisualStudio)' != 'True' or Exists('..\HyperVExtension\src\DevSetupAgent\bin\arm64\($Configuration)\DevSetupAgent_arm64.zip'))">

```

Comment:
Could this possibly be why we're getting the build error for arm64. looks like we're using `($Configuration)\` and not `$(Configuration)\` for arm64  and also for the `or Exists(...` part of both these items.